guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -108,8 +108,8 @@
     "        - labels.csv: 'B-Person', 'I-Person', 'I-Person', 'L-Person', 'O', 'O', 'O', 'U-Country'\n",
```

Comment:
<blockquote>ner-json formatted training data: {"text": "Sir Chandrashekhara Venkata Raman was born in India.","labels": [[0, 33, "Person"], [46, 51, "Country"]]}</blockquote><p> </p><p>Suggestion: Spacing can be made consistent. Can consider the following:</p><p> </p><p>ner-json formatted training data: {"text": "Sir Chandrashekhara Venkata Raman was born in India.", "labels": [[0, 33, "Person"], [46, 51, "Country"]]}</p><p> </p><blockquote>LBIOU formatted training data:</blockquote><blockquote>text.csv: 'Sir', 'Chandrashekhara', 'Venkata', 'Raman' ,'was', 'born', 'in', 'Germany.'</blockquote><blockquote>labels.csv: 'B-Person', 'I-Person', 'I-Person', 'L-Person', 'O', 'O', 'O', 'U-Country'</blockquote><p> </p><p>Suggestion: Spacing can be made consistent. Can consider the following:</p><p> </p><p>text.csv: 'Sir', 'Chandrashekhara', 'Venkata', 'Raman', 'was', 'born', 'in', 'Germany.'</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>